app/views/partials/code.njk

```diff
@@ -3,6 +3,12 @@
 <pre><code class="govuk-!-f-16">{{- componentHtml | e -}}</code></pre>
 
 <h4 class="govuk-heading-s">Macro</h4>
-<pre class="govuk-!-mb-r0 govuk-!-f-16"><code>{% raw %}{{ {% endraw %}{{ componentName | componentNameToMacroName }}(
-  {{- item.data | dump(2) -}}
-  ){% raw %} }}{% endraw %}</code></pre>
+<pre class="govuk-!-mb-r0 govuk-!-f-16">
+  <code>
+    {% raw %}{% from '{% endraw %}{{ componentName }}/macro.njk' import {{ componentName | componentNameToMacroName }} {% raw %}%}{% endraw %}
```

Comment:
Is there a reason the first {% raw %} wraps `from '` as well as the opening tag?

Reply:
Thanks for catching 👍